src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs

```diff
@@ -61,7 +63,7 @@ protected override CodeGenOutput GenerateOutputCore(ServiceInfo service)
 
 					code.WriteLine();
 					WriteJsDoc(code, service);
-					typeNames.Add($"I{capModuleName}");
+					requestTypeNames.Add($"I{capModuleName}");
```

Comment:
This isn't technically a request type, but wherever I needed the request types I also needed this so I included it here.